frontend/src/components/search/SearchBar.tsx

```diff
@@ -1,5 +1,6 @@
 "use client";
 
+import { Icon } from "@trussworks/react-uswds";
```

Comment:
Adding the icon and appropriate classes makes the button responsive so that small screens shows the magnifying glass icon and larger screens shows the word "Search"